fitbenchmarking/results_output.py

```diff
@@ -141,25 +139,33 @@ def build_indiv_linked_problems(results_per_test, group_name):
 
             prev_name = name
             name_index = name + ' ' + str(prob_count)
-            name += ' ' + build_visual_display_page(prob_results, group_name)
+            name = '`' + name + ' ' + build_visual_display_page(prob_results, group_name, results_dir)
             linked_problems.append(name)
 
     return linked_problems
 
 
-def build_visual_display_page(prob_results, group_name):
+def build_visual_display_page(prob_results, group_name, results_dir):
     """
     Builds a page containing details of the best fit for a problem.
     @param prob_results:: the list of results for a problem
     @param group_name :: the name of the group, e.g. "nist_lower"
     """
 
+    # Directory that holds all the visual display pages
+    VDPages_dir = os.path.join(results_dir, "neutron", "VDPages")
```

Comment:
Please change to VDPages_dir to support_pages_dir. Also please use PEP 8 style variable naming convention